.pre-commit-config.yaml

```diff
@@ -5,6 +5,7 @@ repos:
     rev: v3.2.0
     hooks:
     -   id: trailing-whitespace
+        exclude_types: ["csv"]
```

Comment:
I see it as an important point to agree on next meeting on how we want to handle line breaks and what files to exclude. In some of the other PRs I added a rule to exclude all files starting with "expected*" for the same purpose. I think both methods work. If we do not want to add a line break for JSON files then we shall likely exclude all JSON files as well.